scripts/gen-deploy.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
```

Comment:
change file permission, not  able to execute it

Reply:
Done, Windows devstation things :')